src/main/java/httpserver/Main.java

```diff
@@ -1,19 +1,32 @@
 package httpserver;
 
+import httpserver.httpresources.GetWithBody;
+import httpserver.httpresources.GetWithoutBody;
+import httpserver.httpresources.MethodNotAllowed;
 import infrastructure.Listener;
 import infrastructure.Logger;
 import infrastructure.Server;
 
 import java.net.ServerSocket;
-import java.util.concurrent.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
 
 public class Main {
     public static void main(String[] args) throws Exception {
         ServerSocket serverSocket = new ServerSocket(5000);
         Executor executor = Executors.newCachedThreadPool();
         Logger logger = new Logger(System.out);
         HttpPresenter presenter = new HttpPresenter();
-        HttpResponder responder = new HttpResponder(presenter);
+        HttpParser parser = new HttpParser();
+        List<Route> routes = new ArrayList<Route>(Arrays.asList(
+                new Route("/simple_get", new GetWithoutBody()),
+                new Route("/simple_get_with_body", new GetWithBody())
```

Comment:
Manju and I discussed this on our pairing session this afternoon. I suggested that maybe it is premature to this type of router. We also discussed using a List vs a Map.